Controller/Admin/ModuleSupport.php

```diff
@@ -208,16 +207,55 @@ private function _validateRequest()
         $this->_aViewData['fromEmail'] = $sFromEmail;
         $this->_aViewData['body'] = $sBody;
 
-        if (empty($sFromEmail) || !Helper::isEmailValid($sFromEmail)) {
+        if (!$this->_isFromEmailValid($sFromEmail) || !$this->_isReplyToEmailValid($sReplyToEmail)) {
```

Comment:
The reason we implemented it this way is that the `fromEmail` is mandatory while the `replyToEmail` is an optional field. Refactoring it to one single `isEmailValid` would make the `if` statement rather cluttered because we need to include the `empty` and `!empty` checks depending if the field is optional or mandatory.

Reply:
I understand. Thank you. We can keep it this way  then. Please just add a comment explaining this above the if statement.